packages/contentstack-export-to-csv/src/util/index.js

```diff
@@ -678,6 +683,174 @@ function wait(time) {
+  }
+  process.exit(1);
+}
+async function apiRequestHandler(org, queryParam = {}) {
+  const headers = {
+    authtoken: configHandler.get('authtoken'),
+    organization_uid: org.uid,
+    'Content-Type': 'application/json',
+    api_version: 1.1,
+  };
+
+  return await new HttpClient()
+    .headers(headers)
+    .queryParams(queryParam)
+    .get(`${configHandler.get('region')?.cma}/organizations/${org.uid}/teams`)
+    .then((res) => {
+      const { status, data } = res;
+      if(status===200) {
+        return data;
+      } else {
+        cliux.print(`${data.error_message || data.message || data.errorMessage}`, { color: 'red' });
+        process.exit(1);
+      }
+    })
+    .catch((error) => {
+      handleErrorMsg(error);
+    });
+}
+
+async function exportOrgTeams(managementAPIClient, org) {
+  let teamsObjectArray = [];
+  let skip = 0;
+  let limit = config.limit || 100;
+  do {
+    const data = await apiRequestHandler(org, { skip: skip, limit: limit, includeUserDetails: true });
+    skip += limit;
+    teamsObjectArray = [...teamsObjectArray, ...data?.teams];
+    if (skip > data?.count) break;
+  } while (1);
+  teamsObjectArray = await cleanTeamsData(teamsObjectArray, managementAPIClient, org);
+  return teamsObjectArray;
+}
+
+async function getOrgRoles(managementAPIClient, org) {
+  let roleMap = {}; // for org level there are two roles only admin and member
+
+  // SDK call to get the role uids
+  managementAPIClient
+    .organization(org.uid)
+    .roles()
+    .then((roles) => {
+      roles.items.forEach((item) => {
+        if (item.name === 'member' || item.name === 'admin') {
+          roleMap.name = item.uid;
+        }
+      });
+    })
+    .catch((err) => {
+      handleErrorMsg(err);
+    });
+  return roleMap;
+}
+
+async function cleanTeamsData(data, managementAPIClient, org) {
+  const roleMap = await getOrgRoles(managementAPIClient, org);
+  const fieldToBeDeleted = [
+    '_id',
+    'createdAt',
+    'createdBy',
+    'updatedAt',
+    'updatedBy',
+    '__v',
+    'createdByUserName',
+    'updatedByUserName',
+    'organizationUid',
+  ];
+  if (data?.length !== 0) {
+    data.forEach((team) => {
+      fieldToBeDeleted.forEach((fields) => {
+        delete team[fields];
+      });
+      if (!team.hasOwnProperty('description')) {
+        team.description = '';
+      }
+      if (team.organizationRole === roleMap['member']) {
+        team.organizationRole = 'member';
+      } else {
+        team.organizationRole = 'admin';
+      }
+      team.Total_Members = team?.users?.length || 0;
+    });
+  } else {
+    cliux.print(`info: There are no teams in the given org`);
+  }
+  return data;
+}
+
+async function exportTeams(managementAPIClient, organization, teamUid) {
+  const allTeamsData = await exportOrgTeams(managementAPIClient, organization);
+  if (allTeamsData.length === 0) {
+    this.log(`There are not teams in the organization named ${organization?.name}`);
```

Comment:
@cs-raj, Could you pls confirm this.log ? I think in this case it won't work as we're not inherting command class here.
And also instead of if (allTeamsData.length === 0) we can use if (!allTeamsData?.length).

Reply:
Yes this.log gave me the error when i tested with org having no teams therefore removed it and used console.log instead